docs/reference/errors-and-warnings/NU3028.md

```diff
@@ -17,21 +17,23 @@ f1_keywords:
 <pre>The author primary signature's timestamp found a chain building issue: The revocation function was unable to check revocation because the revocation server could not be reached. For more information, visit https://aka.ms/certificateRevocationMode</pre>
 
 ### Issue
+
 Certificate chain building failed for the timestamp signature. The timestamp signing certificate is untrusted, revoked, or revocation information for the certificate is unavailable.
 
 On Windows only, this issue may occur the first time a root certificate is observed.  During certificate chain building, Windows fetches relevant 3rd party root certificates on first use and adds them as locally trusted root certificates.  Internally, Windows initiates this network fetch with an RPC call, and if the system is sufficiently busy, this RPC call may fail.  This failure results in the root certificate not being locally trusted.  For NuGet users, the resulting error is "A certification chain processed correctly but terminated in a root certificate that is not trusted by the trust provider."
 
 ### Solution
+
 Use a trusted and valid certificate. Check internet connectivity.
 
 For Linux and macOS, see [NuGet signed-package verification](/dotnet/core/tools/nuget-signed-package-verification).  Specifically for untrusted root certificate warnings/errors on Linux and macOS, also see [NU3042](NU3042.md).
 
 #### Revocation check mode
+
 > [!Note]
 > This option is available starting from NuGet 4.8.1.
 
-If the machine has restricted internet access (such as a build machine in a CI/CD scenario), installing/restoring a signed nuget package will result in this warning since the revocation servers are not reachable. This is expected.
-However, in some cases, this may have unintended concequences such as the package install/restore taking longer than usual. If that happens, you can work around it by setting the `NUGET_CERT_REVOCATION_MODE` environment variable to `offline`. This will force NuGet to check the revocation status of the certificate only against the cached certificate revocation list, and NuGet will not attempt to reach revocation servers.
+If the machine has restricted internet access (such as a build machine in a CI/CD scenario), installing/restoring a signed nuget package will result in this warning since the revocation servers are not reachable. This is expected. However, in some cases, this may have unintended concequences such as the package install/restore taking longer than usual. If that happens, you can work around it by setting the `NUGET_CERT_REVOCATION_MODE` environment variable to `offline`. This will force NuGet to check the revocation status of the certificate only against the cached certificate revocation list, and NuGet will not attempt to reach revocation servers.
```

Comment:
If my markdown knowledge is up to date, this *won't* change the rendering correct?

Starting a sentence on a new line improves reviewability as it makes the diffs shorter/faster to spot. 

I think we should revert this.